### DISCUSSION TITLE
`not-found` for Group Routes?

### DISCUSSION THREAD

**Discussion by vignesh-gupta** [vercel/next.js#54603] [App Router] [answered]
Hello Guys! 

I'm trying to implement a not-found page for a group route without having a global not-found page. I've tried multiple stuff but still it shows default 404 page. 

I want when I hit `<host>/project213` or `<host>/project2113` to be redirected my my custom 404 user  **(root)/not-found.tsx**  or **(root)/[[...not-found]]/page.tsx**. I read these on stack overflow and Next Discussion tab 

`next info`

```
 Operating System:
      Platform: linux
      Arch: x64
      Version: #28-Ubuntu SMP PREEMPT_DYNAMIC Wed Jul 12 22:39:51 UTC 2023
    Binaries:
      Node: 18.17.1
      npm: 9.6.7
      Yarn: N/A
      pnpm: 8.6.12
    Relevant Packages:
      next: 13.4.19
      eslint-config-next: 13.4.19
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.1.6
    Next.js Config:
      output: N/A

```

My Folder Structure: 
```sh

app
├── favicon.ico
├── globals.css
├── layout.tsx
├── (root)
│   ├── layout.tsx
│   ├── [[...not-found]]
│   │   └── page.tsx
│   ├── not-found.tsx
│   └── page.tsx
└── (studio)
    └── admin
        └── [[...index]]
            └── page.tsx

```

**Comment by devjiwonchoi** [comment]
Place `not-found` inside the route groups. It isn't working for now, but will be resolved by #54931.

**Reply by vignesh-gupta** [reply, marked as answer]
Thanks!

**Reply by flacial** [reply]
This is an old thread, but FYI, the selected answer shouldn't be your response to the actual answer but the answer itself.

**Comment by kmvan** [comment]
```
app
├── [locale]
│   ├── (frontend)
│   │   ├── [[...not-found]]
│   │   ├── layout.tsx
│   │   ├── page.tsx
│   │   ├── not-found.tsx
│   ├── (backend)
│   │   ├── backend
...
```
I got `Error: You cannot define a route with the same specificity as a optional catch-all route ("/[locale]" and "/[locale][[...not-found]]").`
